drivers/peci/peci_ite_it8xxx2.c

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_peci_it8xxx2
+
+#include <drivers/gpio.h>
+#include <drivers/pinmux.h>
+#include <drivers/peci.h>
+#include <kernel.h>
+#include <errno.h>
+#include <device.h>
+#include <drivers/peci.h>
+#include <soc.h>
+#include <soc_dt.h>
+#include <logging/log.h>
+#include <sys/util.h>
+
+LOG_MODULE_REGISTER(peci_ite_it8xxx2, CONFIG_PECI_LOG_LEVEL);
+
+/*
+ * This driver is single-instance. If the devicetree contains multiple
+ * instances, this will fail and the driver needs to be revisited.
+ */
+BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) <= 1,
+	     "Unsupported PECI Instance");
+
```

Comment:
This driver currently requires interrupts, so you should check that `CONFIG_PECI_INTERRUPT_DRIVEN` is enabled.

Reply:
Truly appreciative of your guidance, I'll add it ASAP. :-D

Reply:
Added the following statements:

```
BUILD_ASSERT(IS_ENABLED(CONFIG_PECI_INTERRUPT_DRIVEN),
	     "Please enable the option CONFIG_PECI_INTERRUPT_DRIVEN");
```